### DISCUSSION TITLE
Gradle dependencies are not found in combination with "import"

### DISCUSSION THREAD

**Discussion by moettti** [renovatebot/renovate#25373] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovate/renovate:37.31.5-slim

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Not all dependencies in our gradle (kotlin dsl based) are found and searched for updates.
From the gradle dependencies in build.gradle.kts
`
import ModuleConfiguration.Build.Database
...
   // works
   //runtimeOnly("com.h2database:h2:${ModuleConfiguration.Build.Database.h2Version}")
   // does not work
   runtimeOnly("com.h2database:h2:${Database.h2Version}")
`
The first `runtimeOnly("com.h2database:h2:${ModuleConfiguration.Build.Database.h2Version}")` is found, analysed and leads to a pull request. 
The second `runtimeOnly("com.h2database:h2:${Database.h2Version}")` with the import of the `ModuleConfiguration.Build.Database` is not found.

As a workaround we could eliminate all the imports, but they help us keeping everything clear.

A minimalistic gradle project can be found under https://github.com/moettti/renovate-gradle-issue/blob/main/build.gradle.kts#L34-L37.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by Churro** [comment, marked as answer]
This is currently not supported by the `gradle` manager. I'd suggest to treat it as a feature request, rather than a bug. I haven't entirely thought through the "how" yet but generally it should be feasible to add this functionality.

**Reply by rarkins** [reply]
Please create a feature request with reproduction repo
